Application/App.js

```diff
@@ -24,9 +24,9 @@ export default class App extends Component {
       isAuthenticated: false,
       theme: 'dark',
     };
-
-    //Temprory Solution to remove Timer warning on android
-    YellowBox.ignoreWarnings(['Setting a timer']);
+    //Temprory Solution to remove Timer warning on android, hide virtualized lists error from user, but not console.
+    // Placing a flat list inside scrollview negates the advantages of flatlist so we get this warning.
+    YellowBox.ignoreWarnings(['Setting a timer', 'VirtualizedLists should never be nested']);
```

Comment:
Temporarily hide this from the user, but not from the developer in the console. We get this warning because using scroll view directly with flatlist negates the benefits of the faltlist so we get the warning. The general consensus online I store use SafeAreaView instead of scrollview but it messed up the entire page when I tried replacing it. I was wondering if one of you could try to get SafeAreaView to work in our application instead of ScrollView